ldk/javascript/src/webpack/CheckWarningsAndErrorsPlugin.ts

```diff
@@ -16,6 +16,12 @@ export class CheckLoopOpenHandlerPlugin {
             openHandlerMissingWarning.file = assetName;
             compilation.warnings.push(openHandlerMissingWarning);
           }
+          if (fileContent?.includes('screen.ocr(')) {
+            const ocrDeprecationWarningMsg =
+              '[screen.ocr] ocr will be deprecated soon. Please use listenOcrMonitor instead.';
+            const openHandlerMissingWarning = new WebpackError(ocrDeprecationWarningMsg);
```

Comment:
Just a couple things:

1. This var should be renamed so it's not referencing "openHandler"
2. You could/should probably do something similar to line 16 (setting its `file` property)
3. It would probably be nice to move the `warningMsg` var to be inside the `if` statement on line 14.
4. You probably want to update the plugin name on line 6.

Reply:
Ahhh thx! I just updated according to your comments, please review!

Reply:
Looks great, thanks for updating!